src/emqx_mgmt_cli.erl

```diff
@@ -556,11 +556,23 @@ log(["handlers", "set-level", HandlerId, Level]) ->
             emqx_cli:print("[error] ~p~n", [Error])
     end;
 
+log(["set-level", Level]) ->
+   [case emqx_logger:set_log_handler_level(ID, list_to_atom(Level)) of
+        ok ->
+            {Id, NewLevel, Dst} =emqx_logger:get_log_handler(ID),
+            emqx_cli:print("LogHandler(id=~s, level=~s, destination=~s)~n", [Id, NewLevel, Dst]);
+        {error, Error} ->
+            emqx_cli:print("[error] ~p~n", [Error])
+    end || {ID, _Lev, _Dst} <- emqx_logger:get_log_handlers()],
```

Comment:
When command line `./bin/emqx_ctl log set-level <Level>` is executed,  we need to change all the levels, not only for the log handlers, but also the `primary` log level.

Reply:
Always test your code after having made some changes.